xbmc/ServiceBroker.h

```diff
@@ -10,6 +10,8 @@
 
 #include <memory>
 
+#include "utils/GlobalsHandling.h"
```

Comment:
> Included header files have to be sorted (case sensitive) alphabetically to prevent duplicates and allow better overview, with an empty line clearly separating sections.
>
> Header order has to be:
> - [...]
> - Other Kodi includes (platform independent)
> - [...]
> - C and C++ system files
> - [...]

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#7-headers